packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -480,43 +483,177 @@ export class ResourceService {
     return resource;
   }
 
-  async createResourceEntitiesFromExistingResource(
-    args: CreateResourceEntitiesArgs,
+  async createModelGroupService(
```

Comment:
I dont think this function is the best decision.
we need to use a function that know how to create a service with default settings. it should be generic, and not something specific to the BTM